config.toml

```diff
@@ -33,9 +33,32 @@
    # also (minimum) hysteresis nodes for the minimum number of nodes
    SwitchHysteresisForMinNodesEnableEpoch = 1
 
+   # TransactionSignedWithTxHashEnableEpoch represents the epoch when the node will also accept transactions that are
+   # signed with the hash of transaction
+   TransactionSignedWithTxHashEnableEpoch = 1000000
```

Comment:
1000000?

Reply:
Valid for all questions: yes, we are enabling the first wave without the pressure of the second wave. A binary update will certainly follow, we can enable them then.